awx/main/scheduler/task_manager.py

```diff
@@ -539,6 +552,10 @@ def process_pending_tasks(self, pending_tasks):
                     logger.debug("No instance available in group {} to run job {} w/ capacity requirement {}".format(
                                  rampart_group.name, task.log_format, task.task_impact))
             if not found_acceptable_queue:
+                job_explanation = "This job is not ready to start because there is not enough available capacity."
```

Comment:
This string should be wrapped in i18n `_(...)`

Reply:
I am not sure if this is still relevant https://github.com/ansible/awx/blob/dfa65225d90af717b5011580a4ed35d376dd72e0/awx/api/serializers.py#L806

does this mean we shouldn't mark job_explanation for translation?

Reply:
Hey, good catch!  Yep, I think you're right about this.

Reply:
Well.

Reply:
I think in this case we might *actually* want to wrap this in `ugettext_noop`.  This will *mark* the string for translation (so it shows up in our generated i18n files), but it won't *actually* translate it here.